### DISCUSSION TITLE
Gestion des données et documents topographiques sur Grottocenter

### DISCUSSION THREAD

**Discussion by urien** [GrottoCenter/grottocenter-front#870] [Ideas]
Les données et documents topographiques sont une question très importante lorsque l'on souhaite archiver et partager tout ce qui est lié au monde souterrain.

Nous allons commencer à nous intéresser à ce sujet de manière plus active dans les prochaines semaines, les prochains mois. Cette page permettra d'échanger et de définir une vision commune et des objectifs, une voie pour atteindre ces objectifs.

Je commence à rédiger et vous invite à corriger, compléter, illustrer

# Quelles sont les formes que peuvent prendre les topographies ?

- Des images non modifiables
- Des données correspondant au relevé toographique
- Des données correspondant au relevé réalisé par un LIDAR 
- Des données correspondant au relevé réalisé par photogramétrie
- Un modèle 3D
- Des images 2D éditables

# Quelles sont les actions que nous pourrions mettre en place ?
pour gérer toutes ces données

## 1. Visualiser les documents
- Le navigateur permet directement de visualiser les fichiers image (png, svg, png,...) ainsi que les pdf
- Mettre en place des outils permettant de visualiser sur fond cartographique les données topographiques qui sont dans un format qui le permet (kml par exemple)
- Mettre en place le service existant qui permet de visualiser en 3D les fichiers Thérion au format lox
- Créer un module permettant de visualiser les données concernant les points topos (th, tro,...
- Créer un module permettant de visualiser les données correspondant au dessin des topographies (th2,...)
- Créer un module permettant de visualiser les modèles 3D (fichiers obj)
**Un travail sur ce sujet est proposé aux étudiants Polytech**

## 2. Importer / Exporter des fichiers

Mettre en place des outils permettant d'importer les données depuis d'autres applications, de les transformer dans un autre format, de les archiver dans un format normalisé (à définir dans Karstlink). Ces outils doivent permettre de générer des fichiers pour  visualiser sur la carte à partir des données concernant les points topo ou de produire un pdf à partir d'un dessin topographique,...

Un point "dur" concerne les fichiers de modèles 3D qui sont extrêmement volumineux et qu'il peut être utile de partager en ligne ou de confier à Grottocenter pour réaliser des traitement permettant de générer un modèle numérique simplifié 

## 3. Création / edition

Permettre la création et l'édition en ligne de la topographie : assemblage de données issues de différents fichiers, dessin de la topographie en 2D ou en 3D

## 4. Gestion des points

Permettre de définir des points nommés dans la topographie, ces points pouvant être associés à des observations réalisées dans la cavité

**Comment by Souterweb** [comment]
Penser aussi aux données obtenues par photogrammétrie avec un smartphone par exemple.

**Reply by urien** [reply]
Je te propose de compléter directement le document

**Reply by Souterweb** [reply]
Je ne vois aucun moyen de modifier ton texte (pas de bouton "edit", pas de "crayon"...). Il semble verrouillé et accessible uniquement à toi.
